emails/templates/partials/layout/theme.css

```diff
@@ -0,0 +1,29 @@
+  .canvas {
+    background-color: {{ ternary "#111217" "#FFF" (eq $theme "dark") }};
```

Comment:
We can probably simplify this a bit by introducing another variable

`$wantsDarkTheme := eq $theme "dark"`